config/jobs/kubernetes/k8s-infra-public-pii/k8s-infra-public-pii.yaml

```
@@ -0,0 +1,34 @@
+periodics:
+  - interval: 168h
+    agent: kubernetes
+    name: public-log-vendor-matcher
```

```suggestion
    name: public-log-vendor-matcher
    cluster: k8s-infra-prow-build-trusted
```

This shouldn't be running in the google.com build cluster